src/peer.js

```diff
@@ -22,6 +22,101 @@ function isAllTracksEnded(stream) {
     return isAllTracksEnded;
 }
 
+// takes an mLine, modifies it to put the given
+// codec ID at the front of the list
+function setDefaultCodec(mLine, id) {
+    // m line example format:
+    // m=video 60372 UDP/TLS/RTP/SAVPF 100 101 116 117 96
+    let elements = mLine.split(' ');
+    let newLine = [];
+    let index = 0;
+    for (var i = 0; i < elements.length; i++) {
+        if (index === 3) {
+            newLine[index++] = id;
+        }
+        if (elements[i] !== id || index < 3) {
+            newLine[index++] = elements[i];
+        }
+    }
+    return newLine.join(' ');
+}
+
+/** adds b=as <bitrate> to the sdp */
+function setBitrate(sdp, mediaType, bitrate) {
+    const sdpLines = sdp.split('\r\n');
+    let mLineIndex = null;
+    // find the m line matching our mediatype, so we can make sure
+    // we're modifying the correct section
+    for (var i = 0; i < sdpLines.length; i++) {
+        if (sdpLines[i].search('m=' + mediaType) !== -1) {
+            mLineIndex = i;
+            break;
+        }
+    }
+
+    let index = mLineIndex + 1;
```

Comment:
i think technically it doesn't really matter, because if it doesn't find `mLineIndex` it means the SDP is broken anyway, but i'll add something to handle the case

Reply:
I thought that might be the case, but my review would be remiss if I didn't mention it :-)
Which is why I approved the PR, I didn't think anything I noted "needed" to be addressed.